helm/hpcc/templates/_warnings.tpl

```diff
@@ -0,0 +1,59 @@
+{{/*
+
+---  DO NOT EDIT THIS FILE ---
+
+##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2022 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+##############################################################################
+
+*/}}
+{{/*
+Generate a list of warnings
+*/}}
+{{- define "hpcc.getWarnings" -}}
```

Comment:
I thought it would be easier to manage the warnings separately. I was thinking there would be a fair number of checks that could be added, making the warnings section quite large (possibly, 100s of lines longs).  Also, I didn't want to add to the already large _helpers.tpl.

Reply:
fair enough